### PR TITLE
Minor fixes, refactor DB initialization

### PR DESCRIPTION
**Pull request by axonxorz**

### CODE REVIEW

File: website/auth.py [hunk 1]
```diff
@@ -26,14 +26,12 @@ def login():
             if user:
                 if check_password_hash(user.password, password):
                     flash('successfully logged in', category='success')
-                return redirect(url_for('views.home'))
```
Comment (PKryptonik) [inline website/auth.py:29]: Should be left as `return redirect(url_for('views.home'))`

**Review by PKryptonik — CHANGES_REQUESTED** [review]
review comment in auth.py file

File: website/auth.py (hunk 1 above)
Comment (axonxorz) [reply website/auth.py:29]: Resolved in 0a0c6fb

**Review by PKryptonik — APPROVED** [review]
Moved where the database is defined from __init__.py to models.py along with a few minor adjustments

**Pull request merged**